packages/web/src/scss/components/UNSTABLE_PartnerLogo/_UNSTABLE_PartnerLogo.scss

```diff
@@ -0,0 +1,15 @@
+@use 'theme' as theme;
+@use 'tools';
+
+.UNSTABLE_PartnerLogo {
```

Comment:
Border radius is missing.

<img width="1792" alt="image" src="https://github.com/lmc-eu/spirit-design-system/assets/33354913/854427a7-a9fd-4909-99b4-88644271c0a5">